### PR TITLE
Added post on asynchronous feedback (no comment section yet).

### PR DESCRIPTION
**Pull request by stkent**

### CODE REVIEW

File: _posts/2015-05-05-the-case-for-asynchronous-feedback.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+- **The feedback recipient has time to reflect before a response is required.**
+Even if your constructive criticism ticks all the right boxes (accurate, neutral, well-phrased, actionable, etc.), the recipient may still be caught off-guard by your insights. That's ok, and totally expected. What's awkward is for the recipient to have to _respond_ to you immediately, before they have had time to digest what you have said. Allowing the recipient time alone to examine your suggestions, think back over their past actions, and place your recommendations in their proper context should ultimately lead to much less defensive, and much more productive, discussions.[^2]
+
+- **Asynchronous communication channels are inherently more impersonal.**
+Using a less-personal method of communication may help negative feedback feel like less of a personal 'attack'. While careful wording is probably a better way of dealing with this issue, every little helps.
```
Comment (macklinu) [inline _posts/2015-05-05-the-case-for-asynchronous-feedback.md]: > every little helps

__every little bit helps__, perhaps?

File: _posts/2015-05-05-the-case-for-asynchronous-feedback.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+- **Remote workers are not disadvantaged.**
```
Comment (macklinu) [inline _posts/2015-05-05-the-case-for-asynchronous-feedback.md]: Good point. :+1:

**Comment by macklinu** [comment]
This is a nice exploration of async feedback. No major criticisms, just that one typo.

:shipit:

File: _posts/2015-05-05-the-case-for-asynchronous-feedback.md (hunk 1 above)
Comment (stkent) [inline _posts/2015-05-05-the-case-for-asynchronous-feedback.md]: They seem to be used equally, see [here](http://en.wiktionary.org/wiki/every_little_helps).

I wonder if UK English prefers the short version; one of our major grocery stores uses the slogan:

tesco

**Comment by stkent** [comment]
Thanks for the review! :ship: :scroll: :eyeglasses: :eyes:

File: _posts/2015-05-05-the-case-for-asynchronous-feedback.md (hunk 1 above)
Comment (macklinu) [inline _posts/2015-05-05-the-case-for-asynchronous-feedback.md]: That's cool, didn't know that! Must be the language barrier between us. :smiley_cat:

Comment (stkent) [inline _posts/2015-05-05-the-case-for-asynchronous-feedback.md]: (brownwater) :smile:

**Pull request merged**